### PR TITLE
#160 tokens for minimal button fill set transparent

### PR DESCRIPTION
**Pull request by jamesmelzer**
also corrected internal alias for red and green

### CODE REVIEW

File: library/tokens/tokens.yaml [hunk 1]
```diff
@@ -10,15 +10,15 @@ color :
...
+    default : &color-blue-default   "#0055FF"
   green :
-    default : &color-green-67 "#37A941"
-    light :   &color-green-20 "#BBE9BF"
+    default : &color-green-default  "#37A941"
+    light :   &color-green-20       "#BBE9BF"
   red :
-    default : &color-red-100  "#F40050"
-    light :   &color-red-35   "#FFA6C4"
+    default : &color-red-default    "#F40050"
```
Comment (kevinmpowell) [inline library/tokens/tokens.yaml]: Why the change from the numeric scale to `default` ? I'd expect everything in this "palette" section to use the numeric scale. I noticed there's `-light` up here as well.

* [ ] Change to numeric scale for all "palette" colors - or talk to @kevinmpowell about why not

File: library/tokens/tokens.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ interactive-color :
     glow :    *elevation-color-default
     border :  *color-grey-95
   minimal :
```
Comment (kevinmpowell) [inline library/tokens/tokens.yaml]: Are these truly all "interactive" colors. Or are these mappings only going to be used in button? It's far more interactive tokens than I've ever seen and I question the practical reuse of all of these values across components.

* [ ] Confirm that "interactive" colors are going to be useful cross-component, otherwise push mappings into `buttons.scss` file as component-level `$variables`

Comment (jamesmelzer) [reply library/tokens/tokens.yaml]: I'm reluctant to handle this in the same ticket. 
- Are some of these tokens unique to button? Yes, definitely. 
- Is this more verbose that elsewhere? It's the most specific I've done. 
- Could we devise a scheme for fewer/simpler central tokens that prevents button from referring directly to the palette tokens? I'm not sure. It'll have to be super generic and palette-like (eg `$spirit-interactive-color-white`, `$spirit-interactive-color-blue`, and so on). First pass:

interactive-color :
  focus :     
    outer : *color-blue-30
    inner : *color-blue-100
  text : 
    white : *color-white
    blue : *color-blue-100
    black : *color-black
    disabled : *color-grey-70
  fill : 
    white : *color-white
    primary : *color-blue-100
    primary-light : *color-blue-15
    primary-medium :  *color-blue-30
    disabled : *color-grey-98
    transparent : transparent
  border : 
    secondary : *color-grey-95
    blue :  *color-blue-100

**Comment by jamesmelzer** [comment]
### Token Mapping

$spirit-interactive-color-focus :   INSTEAD: $spirit-interactive-color-focus-outer
$spirit-interactive-color-input-focus :   INSTEAD: $spirit-interactive-color-focus-inner

$spirit-interactive-color-disabled-text :    INSTEAD: $spirit-interactive-color-disabled-foreground
$spirit-interactive-color-disabled-fill :    INSTEAD: $spirit-interactive-color-disabled-background

$spirit-interactive-color-primary-text :    INSTEAD: $spirit-interactive-color-text-on-dark
$spirit-interactive-color-primary-fill :    INSTEAD: $spirit-interactive-color-primary
$spirit-interactive-color-primary-glow :    *elevation-color-blue

$spirit-interactive-color-secondary-text :    INSTEAD: $spirit-interactive-color-text-primary
$spirit-interactive-color-secondary-ill :    INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-secondary-glow :    *elevation-color-default
$spirit-interactive-color-secondary-border :  *color-grey-95

$spirit-interactive-color-minimal-fill :    INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-minimal-fill-hover :   *color-blue-15
$spirit-interactive-color-minimal-fill-active :  *color-blue-30
$spirit-interactive-color-minimal-text :    INSTEAD: $spirit-interactive-color-text-primary

$spirit-interactive-color-on-dark-text :   INSTEAD: $spirit-interactive-color-text-primary
$spirit-interactive-color-on-dark-fill :    INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-on-dark-glow :    *elevation-color-default

$spirit-interactive-color-minimal-on-dark-fill :    INSTEAD: $spirit-interactive-color-primary
$spirit-interactive-color-minimal-on-dark-fill-hover :   INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-minimal-on-dark-fill-active :  INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-minimal-on-dark-text :    INSTEAD: $spirit-interactive-color-text-on-dark
$spirit-interactive-color-minimal-on-dark-text-hover : INSTEAD: $spirit-interactive-color-text-primary
$spirit-interactive-color-minimal-on-dark-text-active : *color-black

$spirit-interactive-color-toggle-text :   INSTEAD: $spirit-interactive-color-text-primary
$spirit-interactive-color-toggle-fill :    INSTEAD: $spirit-interactive-color-on-dark
$spirit-interactive-color-toggle-fill-hover :   *color-blue-15
$spirit-interactive-color-toggle-glow :    *elevation-color-default
$spirit-interactive-color-toggle-border :  INSTEAD: $spirit-interactive-color-primary


$spirit-interactive-color-success-text : INSTEAD: $spirit-interactive-color-text-on-dark
$spirit-interactive-color-success-fill : *color-green-67
$spirit-interactive-color-danger-text : INSTEAD: $spirit-interactive-color-text-on-dark
$spirit-interactive-color-danger-fill : *color-red-100

**Comment by jamesmelzer** [comment]
@kevinmpowell can you take another look at this tokens PR? I'd like to merge this in before Joe returns and starts back up on `button` (which this breaks).

File: library/components/button/button.scss [hunk 3]
```diff
@@ -134,7 +134,7 @@ $spirit-button-transition-timing-function: ease-out;
   display: flex;
   flex-flow: row nowrap;
   justify-content: center;
-  min-width: $spirit-size-icon-buttonwidth;
+  min-width: 130px;
```
Comment (patrickmckowen) [inline library/components/button/button.scss]: @kevinmpowell there are buttons that will be smaller than 130px.  Do we need to define a minimum width?

Comment (kevinmpowell) [reply library/components/button/button.scss]: I didn't dig into the details of why this is here. I just got rid of the token and hard-coded the value. I assume @joesnellpdx will address when he gets back.

Comment (joesnellpdx) [reply library/components/button/button.scss]: @kevinmpowell @patrickmckowen 

That value was there specifically for the following button in the sticker sheet ( last button in buttons.html ):

screen shot 2018-07-16 at 1 22 39 pm
